js_modules/dagster-ui/packages/ui-core/src/assets/AssetCatalogLineage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {useCallback, useMemo} from 'react';
+import {useHistory} from 'react-router';
+
+import {assetDetailsPathForKey} from './assetDetailsPathForKey';
+import {AssetGraphExplorer} from '../asset-graph/AssetGraphExplorer';
+import {AssetGraphViewType} from '../asset-graph/Utils';
+import {AssetLocation} from '../asset-graph/useFindAssetLocation';
+import {useOpenInNewTab} from '../hooks/useOpenInNewTab';
+import {ExplorerPath} from '../pipelines/PipelinePathUtils';
+import {workspacePathFromAddress} from '../workspace/workspacePath';
+
+export const AssetCatalogLineage = React.memo(
```

Comment:
I'm not sure this is really a "Lineage" view in the same sense that `AssetNodeLineage` is a lineage view. It seems like it's AssetCatalogAssetGraph to me?